### PR TITLE
Add connection to database

### PR DESCRIPTION
**Pull request by esoubiran-aneo**
- feat(mongo): add database connection
- feat(mongo): support for production

### CODE REVIEW

**Comment by esoubiran-aneo** [comment]
need to merge #29 to add documentation

File: api.Dockerfile [hunk 1]
```diff
@@ -1,32 +1,32 @@
...
-RUN npm install -g pm2
-
-COPY --from=build /usr/src/app/dist/apps/api ./
-
-RUN npm install
-
-CMD ["pm2-runtime", "start", "./main.js"]
-
-EXPOSE 3333
+FROM node:16.14.2-alpine as build
```
Comment (ngruelaneo) [inline api.Dockerfile:1]: update to 16.15.0

File: app.Dockerfile [hunk 2]
```diff
@@ -1,24 +1,24 @@
...
-FROM nginx:stable as production
-
-WORKDIR /usr/share/nginx/html
-
-RUN rm -rf ./*
-COPY --from=build /usr/src/app/dist/apps/app .
-COPY --from=build /usr/src/app/nginx.conf /etc/nginx/conf.d/default.conf
-
-EXPOSE 80
+FROM node:16.14.2-alpine as build
```
Comment (ngruelaneo) [inline app.Dockerfile:1]: update to 16.15.0

Comment (esoubiran-aneo) [reply app.Dockerfile:1]: Fait dans #38

**Review by ngruelaneo — DISMISSED** [review]

**Review by ngruelaneo — APPROVED** [review]

**Review by ngruelaneo — APPROVED** [review]

**Pull request merged**